index.d.ts

```diff
@@ -0,0 +1,12 @@
+import {Store, Reducer} from 'redux';
+import {Provider} from 'angular2/core';
+
+export function provider(store: Store): Provider;
+
+
+export interface INgRedux extends Store {
```

Comment:
I would vote for creating abstract class instead of interface. What's the deal? TypeScript doesn't retain info about interfaces for runtime so there is no way to inject it like:
```js
constructor(ngRedux: INgRedux) 
```
and you have to do: 
```
constructor(@Inject('ngRedux') ngRedux: INgRedux) 
```

If there would be an abstract class (called preferably `NgRedux`) we could inject it by its type. 

What do you think?